marlowe-marketplace-client/spago.dhall

```diff
@@ -0,0 +1,32 @@
+{-
+Welcome to a Spago project!
+You can edit this file as you like.
+-}
+{ name = "marlowe-marketplace-client"
+, dependencies =
+  [ "aff-promise"
+  , "avar"
+  , "concurrent-queues"
+  , "coroutines"
+  , "debug"
+  , "effect"
+  , "halogen"
+  , "node-fs"
+  , "prelude"
+  , "psci-support"
+  , "remotedata"
+  , "servant-support"
+  , "test-unit"
+  , "undefinable"
+  , "unfoldable"
+  , "uuid"
+  , "web-socket"
```

Comment:
I wanted to build this app without web-socket (as we don't use it here), but the web-commons fail. We could later on create a separate shared lib